backend/src/main/java/com/rota/api/dto/StaffDto.java

```diff
@@ -44,6 +45,10 @@
   @ApiModelProperty("Is the staff currently inactive")
   boolean inactive;
 
+  @ApiModelProperty("The users email address")
+  @Email
```

Comment:
Do we need this verification here or can we assume the email coming in is good?

Reply:
I wouldn't assume anything about user input. If anything, assume that it can be malformed, malicious erc. :slightly_smiling_face: